vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/TextFieldBase.java

```diff
@@ -88,7 +88,7 @@ public int getValueChangeTimeout() {
         return valueChangeTimeout;
     }
 
-    private void applyChangeTimeout() {
+    protected void applyChangeTimeout() {
```

Comment:
Could be changed to package-private if this isn't meant for external use (extensions).

Reply:
Thanks, good point. Updated its visibility modifier.